docs/admin/kubernetes-ops/logs.md

```diff
@@ -3,17 +3,275 @@ tags:
+
+add these changes to your `override.yaml` file:
+
+```yaml
+............
+............
+global:
+  auth-server:
+    appLoggers:
+      authLogLevel: "TRACE"
+      httpLogLevel: "TRACE"
+      persistenceLogLevel: "TRACE"
+      persistenceDurationLogLevel: "TRACE"
+      ldapStatsLogLevel: "TRACE"
+      scriptLogLevel: "TRACE"
+      auditStatsLogLevel: "TRACE"
+............
+............      
+```
+
+Apply the changes: 
+
+```bash
+helm upgrade <helm-release-name> janssen/janssen -f override.yaml -n <namespace>
+```
+
+View the logs of auth-server:
+```bash
+kubectl logs <auth-server-pod> -n <namespace>
+```
+
+
+### config-api
+To get the current log level of config-api:
+
+```bash
+kubectl get configmap -n <namspace> janssen-config-cm -o yaml | grep CN_CONFIG_API_APP_LOGGERS 
+```
+
+Example output:
+```yaml
+CN_CONFIG_API_APP_LOGGERS: 
+'{"
+config_api_log_level":"INFO",
+"config_api_log_target":"STDOUT",
+"ldap_stats_log_level":"INFO",
+"ldap_stats_log_target":"FILE",
+"persistence_duration_log_level":"INFO",
+"persistence_duration_log_target":"FILE",
+"persistence_log_level":"INFO",
+"persistence_log_target":"FILE",
+"script_log_level":"INFO",
+"script_log_target":"FILE"}'
+```
+
+To override the current logging level in config-api, you can either add the desired changes to a yaml file and apply it using `helm`, or
+change it directly using [kubectl edit](#hack-it) command:
+
+- Add changes to yaml:
+
+add these changes to your `override.yaml` file:
+
+```yaml
+............
+............
+global:
+  config-api:
+    appLoggers:
+      configApiLogLevel: "TRACE"
+      persistenceLogLevel: "TRACE"
+      persistenceDurationLogLevel: "TRACE"
+      ldapStatsLogLevel: "TRACE"
+      scriptLogLevel: "TRACE"
+............
+............      
+```
+
+Apply the changes: 
+
+```bash
+helm upgrade <helm-release-name> janssen/janssen -f override.yaml -n <namespace>
+```
+
+View the logs of config-api:
+```bash
+kubectl logs <config-api-pod> -n <namespace>
+```
+
+
+### fido2
+To get the current log level of fido2:
+
+```bash
+kubectl get configmap -n <namespace> janssen-config-cm -o yaml | grep CN_FIDO2_APP_LOGGERS 
+```
+
+Example output:
+```yaml
+CN_FIDO2_APP_LOGGERS: 
+'{"
+fido2_log_level":"INFO",
+"fido2_log_target":"STDOUT",
+"persistence_log_level":"INFO",
+"persistence_log_target":"FILE"}'
+```
+
+
+To override the current logging level in fido2, you can either add the desired changes to a yaml file and apply it using `helm`, or
+change it directly using [kubectl edit](#hack-it) command:
+
+- Add changes to yaml:
+
+add these changes to your `override.yaml` file:
+
+```yaml
+............
+............
+global:
+  fido2:
+    appLoggers:
+      fido2LogLevel: "TRACE"
+      persistenceLogLevel: "TRACE"
+............
+............      
+```
+
+Apply the changes: 
+
+```bash
+helm upgrade <helm-release-name> janssen/janssen -f override.yaml -n <namespace>
+```
+
+View the logs of fido2:
+```bash
+kubectl logs <fido2-pod> -n <namespace>
```

Comment:
to make things easier an get a comprehensive view change this command to `kubectl logs -f deployment/<helm-release-name>-fido2`